.github/workflows/rust.yml

```diff
@@ -134,3 +134,21 @@ jobs:
         cache-on-failure: "true"
     - name: Dry-run publish
       run: cargo publish --dry-run -p ${{ matrix.package }}
+  check-semver:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package:
+        - generic-ec
+        - generic-ec-core
+        # TODO: this fails because the currently published version doesn't compile
+        # by itself. A fix is present and will be published soon, uncomment this
+        # when it's published
+        # - generic-ec-curves
+        - generic-ec-zkp
+    steps:
+    - uses: actions/checkout@v3
+    - name: Check semver
+      uses: obi1kenobi/cargo-semver-checks-action@v2
+      with:
+        package: ${{ matrix.package }}
```

Comment:
It won't fail the pipeline if the version wasn't changed at all? E.g. we don't publish a new version of `core`, so we didn't bump its version, but we still do the semver check, I wonder if it could be a problem

Reply:
Yeah, I have the same question. Can you try it locally?

Reply:
Do we have any published crates that didn't change?

Reply:
Ah never mind, I can just go back in git history

Reply:
Seems to work fine
```
$ cargo semver-checks -p generic-ec
     Parsing generic-ec v0.1.4 (current)
      Parsed [  12.163s] (current)
     Parsing generic-ec v0.1.4 (baseline)
      Parsed [  12.944s] (baseline)
    Checking generic-ec v0.1.4 -> v0.1.4 (no change)
     Checked [   0.013s] 62 checks; 62 passed, 0 unnecessary
    Finished [  25.144s] generic-ec
```

Reply:
Btw I had no problems with installing it, I just had to add cmake to nix-shell